dev-utils/poly-cli/src/static/templates/preview/index.jsx

```diff
@@ -38,6 +38,17 @@ const Footer = (props) => {
 };
 
 const App = () => {
+    const [pod, setPod] = useState(null);
```

Comment:
We should maybe, if possible, add some simple test for this thing, using cypress. That will make easier for everyone to *actually* have tests in features.

Reply:
Agreed. Just not in this PR. I would do it separately.